ipyleaflet/leaflet.py

```diff
@@ -299,7 +300,7 @@ class WMSLayer(TileLayer):
     styles = Unicode().tag(sync=True, o=True)
     format = Unicode('image/jpeg').tag(sync=True, o=True)
     transparent = Bool(False).tag(sync=True, o=True)
-    crs = Enum(values=allowed_crs, default_value='EPSG3857').tag(sync=True)
+    crs = Dict(default_value=projections.EPSG3857).tag(sync=True, o=True)
```

Comment:
You might need to remove `o=True` here as well.

Reply:
I forgot this one, done!